src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
The `fill` method currently allows for any `Machine` to be passed in, but the list may be more specific. Consider using `T` instead of `Machine` for the `value` parameter to ensure type safety: `void fill(List<? super T> machines, T value);`.

Reply:
not fixed